### DISCUSSION TITLE
Cannot connect to Bitcoin core

### DISCUSSION THREAD

**Discussion by freddiemacd** [cryptoadvance/specter-desktop#1167] [Help]
I fresh ubuntu install on a new computer. I downloaded bitcoin core from the snap store and have it fully synced. I have specter desktop v1.3.1 using the app image.

I copy and pasted the data directory path from the snap subfolders using pwd so I know it is typed correctly. It is /home/user/snap/bitcoin-core/common/.bitcoin

It will still not connect. It is a fresh install so there is no bitcoin.conf file. Bitcoin core has a blank wallet.

Any suggestions?

**Comment by ben-kaufman** [comment]
Hi, you need to create a bitcoin.conf and put there `server=1` then restart Bitcoin Core.
